src/HydraAuctionOnchain/MintingPolicies/Auction.hs

```diff
@@ -1,10 +1,19 @@
 module HydraAuctionOnchain.MintingPolicies.Auction
-  ( auctionEscrowTokenName
+  ( allAuctionTokensBurned
+  , auctionEscrowTokenName
   , auctionMetadataTokenName
   , standingBidTokenName
   ) where
 
-import Plutarch.Api.V2 (PTokenName)
+import Plutarch.Api.V1.Value qualified as Value (psingleton)
+import Plutarch.Api.V2
+  ( AmountGuarantees (NonZero)
```

Comment:
Weird that Plutarch doesn't have a `Negative` constructor in `AmountGuarantees`, though I'm not sure what its benefit would be.